### ISSUE TITLE
fzf-tmux from $PATH is ignored when using the vim plugin

### ISSUE THREAD

**Issue by cpakkala** [junegunn/fzf#2521]
- [ x] I have read through the manual page (`man fzf`)
- [x ] I have the latest version of fzf
- [ x] I have searched through the existing issues

## Info

- OS
    - [x ] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce
1) install fzf (which includes fzf-tmux) system-wide on your machine so that it's in your $PATH
2) clone junegunn/fzf repo and source the plugin/fzf.vim with any plugin manager, or do it manually; doesn't matter.
3) ensure fzf-tmux is used for any fzf commands calls in vim:  g:fzf_layout = { 'tmux': '-p80%,80%' }
4) call any fzf command

The result will be that the fzf-tmux script which is called is the one in the cloned repo bin dir, not the system-wide one in your $PATH.  I don't know if this was intentional or not, but it would be nice if the $PATH version of fzf-tmux overrode the version that's relative to the fzf.vim file (../bin/fzf-tmux), as it does in the case of the fzf executable itself, or at least allow a variable to be set to specify the fzf-tmux to be used.

**Comment by junegunn** [comment]
> fzf-tmux script which is called is the one in the cloned repo bin dir

The motivation behind this is to ensure that fzf or fzf-tmux executable is compatible with the Vim plugin. We can't be sure if the one in $PATH supports the latest features required by the Vim plugin.

System-wide packages are not always up-to-date, and some users complained that Vim plugin of fzf was not working only to find out that the binary in their $PATH was outdated. So we decided that the Vim plugin use the one that is shipped with it to avoid the confusion.

Can you elaborate why you want to use fzf-tmux from your package but Vim plugin from the git cloned repo?

**Comment by cpakkala** [comment]
I understand the reasoning and I don't have an argument against it.  My situation is that I manage vim plugins with a vim plugin manager, but I manage my environment with my system package manager.  The fzf-tmux script has to be kept up to date with tmux, which is supplied by my system package manager; thus I also chose to manage fzf/fzf-tmux at the system level as well.  I noticed after upgrading my system that fzf-tmux wasn't working anymore inside of vim (there was recently a tmux change that broke older fzf-tmux scripts).  Eventually figured out that the vim plugin was using it's own version of fzf-tmux.

There are a number of ways I could work around this, to avoid having two fzf installations, but I just figured it would be nice to have the option of using only the fzf.vim without the repo version of fzf and fzf-tmux.

Thanks for the great work, btw.
